crates/title_bar/src/title_bar.rs

```diff
@@ -52,7 +54,13 @@ const BOOK_ONBOARDING: &str = "https://dub.sh/zed-c-onboarding";
 
 actions!(collab, [ToggleUserMenu, ToggleProjectMenu, SwitchBranch]);
 
+pub fn init_settings(cx: &mut App) {
```

Comment:
I think we can inline this one-liner, as there are no usages but the below one.
If we ever need to use this in such form, we can always create it back, with the reason behind it.